test/AlgebraicGeometry/ToricVarieties/toric_blowups.jl

```diff
@@ -31,8 +31,8 @@
   amb1 = domain(bd1)
   
   @testset "Trigger issue of PR3006" begin
-    @test is_complete(amb1)
-    @test is_simplicial(amb1)
+    @test is_complete(amb1) == true
+    @test is_simplicial(amb1) == true
```

Comment:
```suggestion
    @test is_complete(amb1)
    @test is_simplicial(amb1)
```
I am not a fan of all of these `== true` here and further below

Reply:
Yeah, I removed those just now.